content/04.results.md

```diff
@@ -173,13 +175,21 @@ This reveals a large disparity when considering that people with a predicted Eas
 When we again compare _Nature_ articles against last authorship in _Nature_, we observe patterns consistent with the citation analysis with all predicted name origins, except for East Asian and Celtic/English closely matching the predicted name origin rate of last authors in _Nature_ (Figure {@fig:suppfig4}c, dark grey and purple lines).
 We find a similar pattern when we consider _The Guardian_ articles, except we additionally find a depletion of predicted European name origins (Figure {@fig:suppfig4}c, dark grey and light grey lines).
 
+To further understand the source of Celtic/English over-enrichment and East Asian under-enrichment, we selected a subset of quotes from people that were also cited in the news article; again, this analysis could only be completed on _Nature_ articles.
+We found that the disparity was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).
+Next, we identified if the journalist name origin had any effect on quote disparities.
+When considering all quotes, we found that journalists with a predicted East Asian name origin had a higher rate of East Asian quoted speakers (Table {@tbl:table5}).
+However, this disparity was reduced when only considering quotes from people cited within the same news article, and reduced further when considering only citations with a US affiliation (Table {@tbl:table6} and {@tbl:table7}).
```

Comment:
I can't quite tell if the disparity that this refers to is the line 181 disparity or the line 178 disparity. Do we definitely mean disparity here?

Reply:
good point, I edited it to be "We found that the **_under-enrichment of East Asian name origins_**"

Reply:
changed in https://github.com/greenelab/nature_news_manuscript/pull/11/commits/86381c5ff0d5c571b051bbb12fd326d76303b8ba